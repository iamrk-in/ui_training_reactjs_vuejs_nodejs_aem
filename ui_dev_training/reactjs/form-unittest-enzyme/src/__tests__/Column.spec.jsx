import React from "react";
import renderer from "react-test-renderer";

import { mount } from "enzyme";
import enzymeConfig from "../../enzymeConfig";
import Column from "../components/Column";


describe("Container", function() {
    
    it("should render correctly", function() {
        const tree = renderer.create(<Column />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    
    it("should have the correct class name", function() {
        const component = mount(<Column />);
        expect(component.find('div').hasClass('col-md-6')).toBe(true);
    });

});