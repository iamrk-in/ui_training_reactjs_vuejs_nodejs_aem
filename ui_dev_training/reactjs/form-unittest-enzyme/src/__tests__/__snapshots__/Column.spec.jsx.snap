// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should render correctly 1`] = `
<div
  className="col-md-6 offset-md-3"
/>
`;
